source/form.html

```diff
@@ -0,0 +1,184 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Кэт энерджи</title>
+</head>
+
+<body class="page__body">
+
+  <header class="page-header">
+    <a class="page-header__logo-cat">
+      <img src="https://via.placeholder.com/50x50" alt="Кот на логотипе «Cat Energy».">
+    </a>
+    <img class="page-header__logo" src="https://via.placeholder.com/100x50" alt="Логотип «Cat Energy».">
+    <nav class="main-nav">
+      <button class="main-nav__toggle" type="button">
+        <span class="visually-hidden">
+          Открыть меню.
+        </span>
+      </button>
+
+      <ul class="main-nav__list">
+        <li class="main-nav__list-item">
+          <a href="index.html">
+            Главная
+          </a>
+        </li>
+        <li class="main-nav__list-item">
+          <a href="catalog.html">
+            Каталог продукции
+          </a>
+        </li>
+        <li class="main-nav__list-item">
+          <a href="form.html">
+            Подбор программы
+          </a>
+        </li>
+      </ul>
+
+    </nav>
+  </header>
+
+  <main class="page-main">
+    <section class="form">
+      <h2 class="form__title">Подбор программы</h2>
+      <p class="form__text">Заполните анкету, и мы подберем программу питания для вашего кота</p>
+
+      <form action="https://echo.htmlacademy.ru/courses" method="post">
+
+        <fieldset class="form__fieldset">
+          <ul class="form__list">
+            <li class="form__item">
+              <label class="form__label form__field--required" for="cat-name-id">Имя:</label>
```

Comment:
```suggestion
              <label class="form__label form__label--required" for="cat-name-id">Имя:</label>
```
Почитай ещё раз про модификаторы

Reply:
поправить все вхождения